### PR TITLE
Add description to _index.md

### PR DESCRIPTION
**Pull request by jailonso**
### What does this PR do?

Add description to index.md in tagging section
### Motivation

Onboarding challenge
### Preview link

https://docs.datadoghq.com/tagging/

https://docs-staging.datadoghq.com/jaime_alonso/add_description_to_tagging
https://docs-staging.datadoghq.com/jaime_alonso/add_description_to_tagging/tagging

### CODE REVIEW

File: content/en/tagging/_index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with tags
 kind: documentation
+description: "Tags add new dimensions to metrics"
```
Comment (l0k0ms) [inline content/en/tagging/_index.md]:
```suggestion
description: "Learn how to assign and use Tags with Datadog."
```

**Comment by l0k0ms** [comment]
Hello @jailonso, many thanks for this PR 🙇 I left a suggestion over the wording, feel free to merge it when ready.

File: content/en/tagging/_index.md (hunk 1 above)
Comment (jailonso) [reply content/en/tagging/_index.md]: thanks! I like your suggestion. :)

Comment (jailonso) [reply content/en/tagging/_index.md]: Hi,

I don't have permission to merge. I got "code owner review required" message.

Thanks,

**Pull request merged**